packages/editor/src/client/panel/embedEditPanel.tsx

```diff
@@ -66,17 +63,27 @@ export function EmbedEditPanel({value, onClose, onConfirm}: EmbedEditPanel) {
 
         if (iframe) {
           const soundCloudMatch = iframe.src.match(/api.soundcloud.com\/tracks\/([0-9]+)/)
-
           if (soundCloudMatch) {
             const [, trackID] = soundCloudMatch
             setEmbed({type: EmbedType.SoundCloudTrack, trackID})
           } else {
-            setEmbed({
-              type: EmbedType.Other,
-              url: iframe.src,
+            // add attributes if set
+            const setEmbedOther = {
               title: iframe.title,
               width: iframe.width ? parseInt(iframe.width) : undefined,
               height: iframe.height ? parseInt(iframe.height) : undefined
+            }
+            // Add styles if set
+            if (!!iframe.style && iframe.hasAttribute('style')) {
```

Comment:
its just a validation and ensure the style attribute really exist, a double validation

Reply:
it applies the style variable only if the are set (please check out the branch and test it your self, i tried to explain this several times)

Reply:
I don't think you need the double validation. If `!!iframe.style` is true then it has the attribute.

And the reason that you tried to explain this several times is why we need a comment explaining what is happening here. Otherwise someone else will get confused when they reach this code.

Reply:
ok i ll ad comments and remove double check